src/components/application_manager/include/application_manager/commands/hmi/navi_set_video_config_response.h

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2017 Xevo Inc.
```

Comment:
Same here, the company name in the header was requested by our legal department.